package.json

```diff
@@ -13,7 +13,13 @@
     "test": "turbo run test"
   },
   "devDependencies": {
+    "@types/html-minifier": "^4.0.4",
+    "@types/mjml": "^4.7.4",
     "husky": "^8.0.3",
     "turbo": "^1.10.12"
+  },
+  "dependencies": {
+    "html-minifier": "^4.0.0",
+    "mjml": "^4.14.1"
```

Comment:
you should add these new dependencies only in the `cognito-functions` module

Reply:
Check the section [Manage Dependencies](https://github.com/pagopa/developer-portal#manage-dependencies) of the README, there are the examples to show how do it